secrets.yml

```diff
@@ -1,3 +1,3 @@
-OPENTRANS_API_KEY: !var infra-gopass-bgdi/mf-chsdi3/OPENTRANS_API_KEY password
-PGUSER: !var infra-gopass-bgdi/mf-chsdi3/PG PGUSER
-PGPASSWORD: !var infra-gopass-bgdi/mf-chsdi3/PG PGPASSWORD
\ No newline at end of file
+PGUSER: !var /amazon-rds/geodata-master/www-data/user --profile $PROFILE
+PGPASSWORD: !var /amazon-rds/geodata-master/www-data/password --profile $PROFILE
+OPENTRANS_API_KEY: !var /opentransportdata.swiss/swisstopo/tokens/departure_board --profile $PROFILE
```

Comment:
Why not hardcoding the profile here ? Looking at the readme all secrets are on the same profile. It make the usage much more difficult.